crates/rolldown_plugin_reporter/src/lib.rs

```diff
@@ -171,42 +170,53 @@ impl Plugin for ReporterPlugin {
       let mut biggest_compress_size = 0;
 
       let mut log_entries = Vec::with_capacity(args.bundle.len());
-      let mut has_compress_chunk =
-        self.report_compressed_size && self.has_compress_chunk.load(Ordering::Relaxed);
-
-      let mut get_compressed_size = |bytes: &[u8]| -> Option<usize> {
-        if self.report_compressed_size {
-          if self.should_log_info && !has_compress_chunk {
-            if self.is_tty {
-              utils::write_line("computing gzip size (0)...");
-            } else {
-              utils::log_info("computing gzip size...");
-            }
-            has_compress_chunk = true;
-            self.has_rendered_chunk.store(true, Ordering::Release);
-          }
-          utils::compute_gzip_size(bytes).inspect(|_| {
-            let compressed_count = self.compressed_count.fetch_add(1, Ordering::SeqCst);
-            if self.should_log_info && self.is_tty {
-              utils::write_line(&format!(
-                "computing gzip size ({})...",
-                itoa::Buffer::new().format(compressed_count)
-              ));
-            }
-          })
+
+      if self.report_compressed_size && self.should_log_info {
```

Comment:
According to the benchmark, zlib-rs could compress about 1Gb content in 1s, [ref](https://trifectatech.org/blog/zlib-rs-is-faster-than-c/#:~:text=On%20x86_64%20linux%2C%20we%20are%20faster%20for%20some%20of%20the%20compression%20levels%20that%20matter%20most%2C%20about%206%25%20at%20the%20default%20level%20of%206%2C%20and%20over%2010%25%20at%20the%20%22best%20compression%22%20level%209.%20But%20we%27re%20still%20slightly%20slower%20for%20most%20of%20the%20other%20levels%20when%20comparing%20to%20zlib%2Dng.), and also we use rayon to compute the chunk gzip size in parallel now. So in most scenarios, the computation phase would be instant, so it would be unnecessary to report the progress anymore.